### PR TITLE
Production overrides

### PR DESCRIPTION
**Pull request by ivan-c**
- Add restart policy
- Add docker healthchecks
  - Hapi: http check against [FHIR capability statement](https://www.hl7.org/fhir/CapabilityStatement.html)
  - postgres: connection check against database

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ Configuration for CNICS services. Includes components for transforming CNICS rel
 
 
 ## Setup
-Clone this repo to your desired location and follow Setup steps in [`dev` directory](./dev/README.md), for deploying a dev environment
+Clone this repo to your desired location and follow Setup steps in [`dev` directory](./dev/README.md), to setup a development deploy, or [`prod`](./prod/README.md) for a production deploy.
```
Comment (pbugni) [inline README.md]:
```suggestion
Clone this repo to your desired location and follow Setup steps in [`dev`](./dev/README.md), to setup a development deploy, or [`prod`](./prod/README.md) for a production deploy.
```

File: prod/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+version: "3.4"
+services:
+  fhir:
+    restart: unless-stopped
+    healthcheck:
+      test:
+        wget --spider --quiet
+          http://localhost:8080/fhir/metadata
```
Comment (pbugni) [inline prod/docker-compose.yaml:9]: nice!

**Review by pbugni — APPROVED** [review]
Well done, thank you.  (one minor nit)

**Review by mcjustin — APPROVED** [review]

**Review by jhowe-uw — COMMENTED** [review]
Missing restart for carl?

File: prod/docker-compose.yaml (hunk 2 above)
Comment (ivan-c) [reply prod/docker-compose.yaml:9]: seems to work well enough for now. When I intentionally misconfigured the DB connection, `dc ps` was correctly showing `unhealthy`

**Comment by ivan-c** [comment]
> Missing restart for carl?

@jhowe-uw 

I think @pbugni and @jsibley have been [running one-off containers via `docker-compose run carl`](https://github.com/uwcirg/carl/#how-to-run), so there's no persistent service

**Comment by jsibley** [comment]
Yes, it's true, no persistent carl container...

**Review by jsibley — APPROVED** [review]
Thanks @ivan-c , looks good!

**Pull request merged**